tests/fail.cmake

```diff
@@ -0,0 +1,2 @@
+set(TOIT_FAILING_TESTS
```

Comment:
Should we have copyright headers in these files too? tests/CMakeLists.txt has such a header.

Reply:
done.